services/efs-guardian/docker/entrypoint.sh

```diff
@@ -88,7 +88,7 @@ if stat $DOCKER_MOUNT >/dev/null 2>&1; then
 fi
 
 echo "$INFO Starting $* ..."
-echo "  $EFS_USER_NAME rights    : $(id "$EFS_USER_NAME")"
+echo "  $(whoami) rights    : $(id $whoami))"
 echo "  local dir : $(ls -al)"
 
-exec gosu "$EFS_USER_NAME" "$@"
+exec "$@"
```

Comment:
I will state this as well here. Do not agree with this approach. But I do not want to block the entire PR. Since a solution to replace this will hopefully be coming up.

Reply:
Yes, I have presented all the arguments I have for now. I am deciding that we will proceed this way so that we can develop the feature quickly and get fast feedback from inhouse users only! We will improve the security design as soon as possible (possibly by running just a specific script with privilege rights).